src/e2e/java/teammates/e2e/util/LNPSpecification.java

```diff
@@ -0,0 +1,59 @@
+package teammates.e2e.util;
+
+/**
+ * Stores the specifications for a LNP test, and verifies whether the results meet the criteria.
+ */
+public class LNPSpecification {
+
+    private double errorRateLimit;
+    private double meanResTimeLimit;
+    private String resultsErrorMessage = "";
+
+    /**
+     * Create a specification class with the specified limits.
+     * @param errorRateLimit Maximum allowable threshold for the percentage of failed requests
+     *                       (0 to 100) to the test endpoint.
+     * @param meanResTimeLimit Maximum allowable threshold for the mean response time
+     *                         (in seconds) for the test endpoint.
+     */
+    public LNPSpecification(double errorRateLimit, double meanResTimeLimit) {
```

Comment:
It would be better to use builder pattern to construct a specification.

Reply:
I've added in a comment regarding this for future reference.